src/iperf_pthread.c

```diff
@@ -6,19 +6,10 @@
  * as Android NDK does not support `pthread_cancel()`.
  */
 
+#include <string.h>
 #include <signal.h>
 #include "iperf_pthread.h"
 
-int pthread_setcanceltype(int type, int *oldtype) { return 0; }
-int pthread_setcancelstate(int state, int *oldstate) { return 0; }
-int pthread_cancel(pthread_t thread_id) {
-    int status;
-    if ((status = iperf_set_thread_exit_handler()) == 0) {
```

Comment:
error: call to undeclared function 'iperf_set_thread_exit_handler'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]